samples/containers/android/src/main/java/com/squareup/sample/container/SampleContainers.kt

```diff
@@ -1,12 +1,11 @@
 package com.squareup.sample.container
 
 import com.squareup.sample.container.overviewdetail.OverviewDetailContainer
-import com.squareup.sample.container.panel.PanelContainer
-import com.squareup.sample.container.panel.ScrimContainer
-import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.sample.container.panel.PanelOverlayDialogFactory
 import com.squareup.workflow1.ui.ViewRegistry
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
 
 @OptIn(WorkflowUiExperimentalApi::class)
 val SampleContainers = ViewRegistry(
-    BackButtonViewFactory, OverviewDetailContainer, PanelContainer, ScrimContainer
+  BackButtonViewFactory, OverviewDetailContainer, PanelOverlayDialogFactory, ScrimContainer
```

Comment:
Oh! I thought `PanelContainerScreen` was more broadly used, but it was actually only in the Tic Tac Toe sample.

- [ ] delete it

Reply:
And maybe the old scrim screen too.